markdown_lab/core/config.py

```
@@ -0,0 +1,200 @@
+# Global default configuration instance
+DEFAULT_CONFIG = MarkdownLabConfig()
+
+
+def get_config() -> MarkdownLabConfig:
+    """Get the current global configuration."""
+    return DEFAULT_CONFIG
+
+
+def set_config(config: MarkdownLabConfig) -> None:
+    """Set the global configuration."""
+    global DEFAULT_CONFIG
```

Ruff (0.11.9)

Using the global statement to update `DEFAULT_CONFIG` is discouraged

(PLW0603)